build/README.md

```diff
@@ -375,3 +379,20 @@ Use TAG to specify the image to transfer into minikube
 ## Dependencies
 
 This project uses the [dep](https://github.com/golang/dep) as a dependency manager. You can see the list of dependencies [here](https://github.com/GoogleCloudPlatform/agones/blob/master/Gopkg.toml).
+
+## Troubleshooting
+
+Frequent issues and possible solutions
+
+#### $GOPATH/$GOROOT error when building
```

Comment:
Do we want to add something specific about how it should be mounted on `/c/` -- or that to check if the mount is occuring correctly (with some troubleshooting steps?)

@fooock did you have some suggestions here?

Reply:
I would indicate that if the project is on a disk other than `c`, it would be necessary to mount the disk on which the project is located. 

Docker for windows automatically assembles disk `c`, but if our project is in `d`, we will have to mount disk `d`.